.travis.yml

```diff
@@ -6,14 +6,16 @@ node_js:
     - "4.1"
 
 before_install:
+    - rvm install 2.2.2
```

Comment:
It does work! Though the key is still "language", not "languages" - my mistake.

https://github.com/edx/edx-analytics-dashboard/blob/master/.travis.yml#L1-L5